build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
 
     implementation 'net.logstash.logback:logstash-logback-encoder:7.2'
 
-    implementation 'uk.gov.digital.ho.hocs:hocs-ukvi-complaint-schema:1.1.1'
+    implementation 'uk.gov.digital.ho.hocs:hocs-ukvi-complaint-schema:186f0b005b206df2cc7c9b8003949fd5a76a5bd8'
```

Comment:
Before moving to GH Packages we were creating GH Releases with semver version numbers. We should look to see if we can continue with that rather than using shas

Reply:
Looks like by default packages are created with the latest commit sha: https://github.com/UKHomeOffice/hocs-ukvi-complaint-schema/packages/1454588

Reply:
That’s a side effect of the smoke build as you cannot flag latest should only be a specific format. We should enforce a semver only in main/ delete SHA packages after smoketest(ing) to ensure latest is at the last release.

Reply:
Thanks for the feedback. I think I get it. So, the current value is what we used for the smoke testing but it needs to be updated to the one from the latest commit e.g. 1454588?

Reply:
Rather than a SHA we should create a version which follows semver convention and use the version number here

Reply:
So when smoke testing a build is generated with the commit SHA - which is what you currently have here and allows you to test your changes. 

On merging the schema change there will be a new semver version - 1.2.0 will be generated based on the labels which you could replace that with here.

Reply:
Okay yep. I get it. Merge the schema then that will let us know the semver